static/images/help.svg

```diff
@@ -0,0 +1,8 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Comment:
Nice. 😎

Reply:
Haha neat, hadn't even seen that - just copied it from the Firefox source code :)